jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -129,27 +138,26 @@ private TestPlatformConfiguration(@Nullable String architecture, @Nullable Strin
 
   @Before
   public void setUp() throws IOException, InvalidImageReferenceException, InferredAuthException {
-    Mockito.when(rawConfiguration.getFromAuth()).thenReturn(authProperty);
-    Mockito.when(rawConfiguration.getEntrypoint()).thenReturn(Optional.empty());
-    Mockito.when(rawConfiguration.getAppRoot()).thenReturn("/app");
+    when(rawConfiguration.getFromAuth()).thenReturn(authProperty);
+    when(rawConfiguration.getEntrypoint()).thenReturn(Optional.empty());
+    when(rawConfiguration.getAppRoot()).thenReturn("/app");
     Mockito.<List<?>>when(rawConfiguration.getPlatforms())
```

Comment:
hm what happens if we just call `when` in line 135?

Reply:
Looks like type information is needed to resolve an incompatible generic type error.